modules/axhal/src/platform/qemu_virt_loongarch64/misc.rs

```diff
@@ -0,0 +1,9 @@
+/// Shutdown the whole system, including all CPUs.
+pub fn terminate() -> ! {
```

Comment:
Is there any instruction to exit QEMU? This is necessary for CI.

Reply:
I didn't find relevant information on the Internet. Maybe I can ask the engineers at la.

Reply:
Mainline qemu does not have an exit function. Is it possible to use other means to exit qemu from the outside?